packages/atomic/src/components/commerce/product-template-components/atomic-product-text/atomic-product-text.tsx

```diff
@@ -51,6 +83,10 @@ export class AtomicProductText
     }
   }
 
+  public isFieldSupportedForHighlighting() {
+    return ['ec_name', 'excerpt'].includes(this.field);
+  }
+
   public render() {
     const productValueAsString = getStringValueFromProductOrNull(
```

Comment:
Not sure how I missed that when I made this component but this conditional is useless so you could remove it
```typescript
productValueAsString ?? this.default,
```